runtime/browser/runtime_url_request_context_getter.cc

```diff
@@ -139,7 +159,7 @@ net::URLRequestContext* RuntimeURLRequestContextGetter::GetURLRequestContext() {
             net::DISK_CACHE,
             net::CACHE_BACKEND_DEFAULT,
             cache_path,
-            0,
+            GetDiskCacheSize(),
```

Comment:
This means you are repeatedly parsing the command line and trying to convert the same string, over and over again. You should do this once during startup and then use a cached value instead.